src/Test/Utilities/Desktop/SigningTestHelpers.cs

```diff
@@ -55,16 +66,14 @@ private unsafe static void InstallKey(byte[] keyBlob, string keyName)
             }
         }
 
-        internal sealed class VirtualizedStrongNameProvider : DesktopStrongNameProvider
+        internal sealed class VirtualizedIOOperations: IStrongNameFileSystem
```

Comment:
I liked the old name better 🤷‍♂️

Reply:
The behavior changed significantly. 

Previously it was extending the StrongNameProvider directly.  

Now, because it needs to work with Portable _and_ Desktop providers, it extends the IStrongNameFIleSystem.

However, I do think that the name could be better.  I've changed it to `VirtualizedStrongNameFileSystem`.